test/float8/test_auto_filter.py

```diff
@@ -0,0 +1,133 @@
+from functools import partial
+
+import pytest
+import torch.nn as nn
+
+from torchao.float8 import _auto_filter_for_recipe
+from torchao.float8.config import Float8LinearRecipeName
+from torchao.float8.float8_linear_utils import (
+    _auto_filter_for_rowwise,
+    _auto_filter_for_tensorwise,
+)
+
+
+@pytest.fixture
+def sample_filter_fqns():
+    """Fixture providing sample filter FQNs."""
+    return ["layer1", "layer2", "filtered_layer"]
+
+
+def test_tensorwise_recipe_returns_partial_function(sample_filter_fqns):
```

Comment:
optional nit: I don't think you need 100% coverage, this behavior seems obvious from the implementation so not required to test it